ingest/rules/fetch_from_ncbi.smk

```diff
@@ -88,12 +88,12 @@ rule format_ncbi_dataset_report:
             --package {input.dataset_package} \
             --fields {params.ncbi_datasets_fields:q} \
             --elide-header \
-            | csvtk fix-quotes -Ht \
-            | csvtk add-header -t -l -n {params.ncbi_datasets_fields:q} \
+            | csv2tsv --csv-delim $'\t' \
```

Comment:
I believe `csvtk fix-quotes` needs to stay because NCBI produces IANA-TSVs while `csvtk` expects CSV-like TSVs. 

(_additional nit_)
Might be simpler to avoid `tsv-select` here as well (e.g. [format_ncbi_dataset_report](https://github.com/nextstrain/pathogen-repo-guide/blob/b4001735fa55448f1426ff90cca93c667ca121ce/ingest/rules/fetch_from_ncbi.smk#L99-L102) in pathogen-repo-guide)